files/mcollective/pluginpackager/templates/aiomodule/README.md.erb

```diff
@@ -10,7 +10,8 @@
 
 <%= @plugin.metadata[:description] %>
 
-The <%= module_name %> module is based on the source from <%= @plugin.metadata[:url] %>.
+The <%= module_name %> module generated automatically, based on the source from <%= @plugin.metadata[:url] %>.
+You can find templates used for generation in https://github.com/choria-io/puppet-mcollective/tree/master/files/mcollective/pluginpackager/templates/aiomodule.
```

Comment:
I understand now. Yeah, your description should be fine, it's not perfect, but at least it's pointing to the right direction.

Later on, I guess it would be nice to have separated repo with for mcollective package plugin, which will be able to package itself, and this should contain all those templates. And then we can link directly to this plugin.

I will update the PR.